e2e/build_uki_test.go

```diff
@@ -56,19 +52,49 @@ var _ = Describe("build-uki", func() {
 		})
 	})
 
-	It("successfully builds a uki iso from a container image", func() {
-		out, err := enki.Run("genkey", "--output", keysDir, "TEST")
-		Expect(err).ToNot(HaveOccurred(), out)
-		fmt.Println(out)
-
-		out, err = enki.Run("build-uki", image, "--output-dir", resultDir, "-k", keysDir, "--output-type", "iso")
-		Expect(err).ToNot(HaveOccurred(), out)
-
-		By("building the iso")
-		_, err = os.Stat(resultFile)
-		Expect(err).ToNot(HaveOccurred())
+	Describe("building an iso", func() {
+		When("secure-boot-enroll is not set", func() {
```

Comment:
IMO having these `When` sections but not putting a differenciation between the other `When` brings little value. We could just have an `It("successfully builds a uki iso from a container image when secure-boot-enrool is not set`

So for a more Ginkgo approach I think we should use a JustBeforeEach here to set those parameters